recipes/mapd-core-cpu/build.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+set -ex
+
+env
+
+ls -la $BUILD_PREFIX/bin
```

Comment:
Why do you need this step? It is quit unorthodox.

Reply:
Yeah, agreed, please remove this line, and the `env` above it.  I am guessing it was put here for debugging

Reply:
Removed (the remnant of debugging).